src/GenAPI/Microsoft.DotNet.GenAPI.Tool/Program.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+
+namespace Microsoft.DotNet.GenAPI.Tool
+{
+    /// <summary>
+    /// CLI frontend for the Roslyn-based GenAPI.
+    /// </summary>
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            // Global options
+            Option<string[]> assembliesOption = new("--assembly",
+                description: "The path to one or more assemblies or directories with assemblies.",
+                parseArgument: ParseAssemblyArgument)
+            {
+                Arity = ArgumentArity.OneOrMore,
+                IsRequired = true
+            };
+
+            Option<string[]?> assemblyReferencesOption = new("--assembly-reference",
+                description: "Paths to assembly references or their underlying directories for a specific target framework in the package.",
+                parseArgument: ParseAssemblyArgument)
+            {
+                Arity = ArgumentArity.ZeroOrMore
+            };
+
+            Option<string[]?> excludeAttributesFilesOption = new("--exclude-attributes-file",
+                description: "The path to one or more attribute exclusion files with types in DocId format.",
+                parseArgument: ParseAssemblyArgument)
```

Comment:
nit: I don' think that we want the assembly parsing logic for attributes files. It's very rare that more than one or two exclusion files are passed in. In such cases, the consumer would just pass them by specifying the option multiple times: `--exclude-attributes-file a.txt --exclude-attributes-file b.txt`.

```suggestion
            Option<string[]?> excludeAttributesFilesOption = new("--exclude-attributes-file",
                "The path to one or more attribute exclusion files with types in DocId format.");
```

Reply:
Let's have the same behavior for all `plural` options